clientd3d/dialog.c

```diff
@@ -734,3 +733,12 @@ void AbortGameDialogs(void)
 	AbortPasswordDialog();
 	AbortPreferencesDialog();
 }
+/************************************************************************/
+/*
+* SafeDialogBoxParam:  A wrapper for DialogBoxParam that resets tooltips when a dialog is opened.
+*/
+INT_PTR __stdcall SafeDialogBoxParam(HINSTANCE hInstance, LPCSTR lpTemplate, HWND hWndParent, DLGPROC lpDialogFunc, LPARAM dwInitParam)
```

Comment:
I don't think we need __stdcall on our own functions.  Win32 functions are declared this way for historical reasons.

Reply:
It appears to be necessary, otherwise we see a bunch of these when compiling:
```
error LNK2001: unresolved external symbol _SafeDialogBoxParam@20
```

Reply:
Can you try M59EXPORT instead?  See other exported functions.

Reply:
This results in the same linker errors. Since we’re replacing a Windows API function, I guess we need to match its calling convention.

Reply:
Hm, no, there's nothing special about this new function.  Where do you get the linker error, and does it persist if you build clean (client and modules) first?